Doc/using/cmdline.rst

```diff
@@ -994,3 +994,71 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
+
+Finding modules
+---------------
+
+A module search path is initialized when Python starts. This module search path
+may be accessed at :data:`sys.path`.
+
+The first entry in the module search path is the directory containing the input
+script (or the the current directory when no file is specified).
```

Comment:
```suggestion
The first entry in the module search path is the directory that contains the
input script, if there is one. Otherwise, the first entry is the current
directory, which is the case when executing the REPL shell, a ``-c`` command,
or ``-m`` module.
```

Reply:
I'd change "REPL shell" to "interactive shell", but otherwise, good catch.

Reply:
Amended, thank you